clients/mobile/src/authentication/otp/hooks/useOtp.ts

```diff
@@ -34,20 +34,28 @@ export const useOtp = (): any => {
     return otp.join('').trim().length !== 6
   }, [otp])
 
+  const handleRegister = async(): Promise<void> => {
+    const isSucessRegister = await submitOtp(email, otp.join(''))
+    if (isSucessRegister) {
```

Comment:
could fail first be a better approach?

Reply:
Updated  [Commit](https://github.com/CraftsmenLtd/BloodConnect/pull/202/commits/ba27593e9bda7d902822fc85881238bf5b32500a)